pages/common/serverless.md

```diff
@@ -0,0 +1,28 @@
+# serverless
+
+> Toolkit for deploying and operating serverless architectures on AWS, Google Cloud, Azure and IBM OpenWhisk.
+> Commands can be run either using the `serverless` command or it's alias, `sls`.
+
+- Create a serverless project:
+
+`serverless create`
+
+- Create a serverless project from a template:
+
+`serverless create --template {{template_name}}`
+
+- Deploy to a cloud provider:
+
+`serverless deploy`
+
+- Display information about a serverless project:
+
+`serverless info`
+
+- Follow the logs for a project:
+
+`serverless logs -t`
+
+- Display metrics for a function:
```

Comment:
Should we replace this by the invoke function command ? Seems to me that invoking a function is more important that displaying metrics. Thoughts ?

Reply:
Hmm, that's an interesting thought. I certainly find myself using invoke more during development but metrics once something is live. You're probably right though in that in general invoking is more important, I'll switch it out.